webpack.config.js

```diff
@@ -5,7 +5,7 @@ const HTMLWebpackPlugin = require('html-webpack-plugin');
 const makeBabelConfig = require('@dhis2/cli-app-scripts/config/makeBabelConfig.js');
 
 const defaultBaseUrl =
-    process.env.NODE_ENV === 'production' ? '..' : 'http://localhost:8080';
+    process.env.NODE_ENV === 'production' ? '..' : 'https://test.e2e.dhis2.org/analytics-2.39';
```

Comment:
Did you intend for these changes in webpack to be committed to the PR branch?

Reply:
No, had to make these changes to locally for testing. I revert them. Thanks.